nncf/experimental/onnx/graph/nncf_graph_builder.py

```diff
@@ -215,3 +218,8 @@ def create_nncf_graph(onnx_model: ModelProto) -> NNCFGraph:
         GraphConverter._add_nncf_input_nodes(onnx_graph, nncf_graph)
         GraphConverter._add_nncf_output_nodes(onnx_graph, nncf_graph)
         return nncf_graph
+
+class ExtendedLayerAttributes(BaseLayerAttributes):
```

Comment:
At the first, I would like to understand whether we need to use these attributes in principle in the general part of the algorithm. Probably, we can use NNCFGraph to specify command in the common part of algorithm. Let's discuss it.

Reply:
I don't see a problem but would call it `ONNXLayerAttributes` just to show that it has ONNX specific but is not something generic.

Reply:
This class is used in the common part of the fast bias correction algorithm. If you rename it to ```ONNXLayerAttributes``` than you can not use it in the common part.

Reply:
This extension would not have arisen if the edges contained the real tensor names. The other way to avoid this problem is to create a sub-model that is needed for the algorithm via model building instead of extraction.